proposals/js-string-builtins/Overview.md

```diff
@@ -632,6 +632,7 @@ func decodeStringFromUTF8Array(
 /// Traps if the length of the UTF-8 encoded string doesn't fit into an i32
 func measureStringAsUTF8(
   string: externref,
+  start: i32
```

Comment:
Do you actually need `start` for your use cases? It isn't used below; so at any rate this PR seems to be incomplete; and the obvious alternative would be to delete the comment and (currently-useless) `start >>>= 0` operation instead.
Considering that [`TextEncoder.encode`](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder/encode) doesn't take a `start` parameter, and this proposal aims to limit itself to repackaging existing functionality, it's not obvious how a `start` parameter would be integrated into the specified algorithm.

Reply:
I don't need `start`, I just saw it referenced in the function body and in some of the other functions so I thought it should be added. I'm OK with removing it as well. I'll update.